pkg/api/customization/workload/workload.go

```diff
@@ -224,6 +239,11 @@ func Formatter(apiContext *types.APIContext, resource *types.RawResource) {
 	resource.Links["self"] = apiContext.URLBuilder.ResourceLinkByID(workloadSchema, workloadID)
 	resource.Links["remove"] = apiContext.URLBuilder.ResourceLinkByID(workloadSchema, workloadID)
 	resource.Links["update"] = apiContext.URLBuilder.ResourceLinkByID(workloadSchema, workloadID)
+	resourceType := resource.Type
+	//add redeploy action to the workload types that support redeploy
```

Comment:
maybe nit: can we have a map of workloads that allow redeploy instead and check with that?